data/hai/hai outfits.txt

```diff
@@ -777,6 +777,7 @@ outfit "Shield Beetle Pendant"
 	thumbnail "outfit/crystal pendant"
 	"quantum keystone" 1
 	description "A pendant made of the same crystalline rocks found on the Hai homeworld, fashioned after a Shield Beetle. A gift from Nanachi."
+	description "	This pendant comes with a finely crafted lit shadowbox so it can be installed and displayed with pride. A trio of mounting brackets - miniatures of those used on Quantum Keystones - provide a secure anchor to keep the pendant safe."
```

Comment:
Ok now I understand the brackets thing.
Might just be me, but `This pendant` seemed a bit awkward, right after a short line that already starts with pendant. Maybe switching to `It`? Not sure if there's a better replacement.
```suggestion
	description "	It comes with a finely crafted lit shadowbox so it can be installed and displayed with pride. A trio of mounting brackets - miniatures of those used on Quantum Keystones - provide a secure anchor to keep the pendant safe."
```

Reply:
The main reason for explicitly mentioning the brackets here is because - unlike the keystone image - the pendant doesn't visibly have them; and comes with the expectation that the player is wearing it. Also, the keystone is a crystal - I personally consider it sensible to assume there's *something* to mount it, as otherwise it'd just be bouncing around or flying off into space. Very few of our outfits explicitly mention mounting hardware of any kind. 

Now, if people are assuming the keystone is literally just another trinket like the pendant, some kind of lucky coin or something to stuff in a pocket... Then yeah, the description needs to be adjusted to clarify that too.